uwblueprint/community-fridge-kw
# feat: email volunteer after admin deletes schedule

## Brief description. What is this change? 

### [Send Volunteer Email When Admin Cancels Donation](https://www.notion.so/uwblueprintexecs/Send-Volunteer-Email-when-Admin-Cancels-Donation-242c92f6c5e8411690f792e5bcd0d965)



## Implementation description. How did you make this change?
* modified `sendSchedulingCancellationEmail` to include sending a email to volunteer if admin deletes donation they signed up for
<img width="866" alt="Screen Shot 2022-04-15 at 5 14 30 PM" src="https://user-images.githubusercontent.com/53016529/163632572-52854d87-b1b0-47f9-9725-e81219117a56.png">



## Steps to test
1. Log in as a donor and create a recurring donation (i. May 1 - May 31st Weekly @ 9pm, Volunteer required)
2. Log in as volunteer A and sign up for a shift for that donation. (i.e May 1)
3. Log in as volunteer B and sign up for a shift for that donation. (i.e May 8)
4. Log in as a donor and cancel the recurring donations.
5. Both volunteer A and B should receive the following cancellation notice email: 

<img width="1102" alt="Screen Shot 2022-04-17 at 4 11 44 PM" src="https://user-images.githubusercontent.com/37950626/163730592-220ebd79-e2ca-4c19-b0f3-1db59a9be221.png">

6. Make sure this works for deleting a single shift (not recurring) as well. 
7. Also make sure that all the confirmation emails are being sent correctly throughout the process.


## Checklist
